### ISSUE TITLE
Requesting same video multiple times causes fatal error

### ISSUE THREAD

**Issue by jupjohn** [pablouser1/ProxiTok#42]
When accessing a video and reloading after initialization, the site fails with the following exception:

```
Fatal error: Uncaught Error: Typed property TikScraper\Items\Video::$item must not be accessed before initialization in /var/www/html/vendor/pablouser1/tikscraper/src/Items/Video.php:50 
Stack trace: 
  #0 /var/www/html/app/Controllers/UserController.php(34): TikScraper\Items\Video->feed() 
  #1 [internal function]: App\Controllers\UserController::video('jathnjoestar', '705960443100388...') 
  #2 /var/www/html/vendor/bramus/router/src/Bramus/Router/Router.php(476): forward_static_call_array(Array, Array)
  #3 /var/www/html/vendor/bramus/router/src/Bramus/Router/Router.php(440): Bramus\Router\Router->invoke('UserController@...', Array)
  #4 /var/www/html/vendor/bramus/router/src/Bramus/Router/Router.php(287): Bramus\Router\Router->handle(Array, true)
  #5 /var/www/html/index.php(17): Bramus\Router\Router->run()
  #6 {main} thrown in /var/www/html/vendor/pablouser1/tikscraper/src/Items/Video.php on line 50
```

Using release 2.3.0.1 after a fresh install, observed on old instance as well.

**Comment by pablouser1** [comment]
Thanks for opening this issue! It is now fixed e1fbf7528181021f73a332f6631fc10c14d1dc31. I've tested it on the official instance and is now working

**Comment by jammehcow** [comment]
Thanks!
